src/ngraph/runtime/cpu/CMakeLists.txt

```diff
@@ -156,9 +156,42 @@ if (NGRAPH_MLIR_ENABLE)
         )
 endif()
 
+set(NGRAPH_CPU_ALL_DATATYPES
+    boolean
+    f32
+    f64
+    i8
+    i16
+    i32
+    i64
+    u8
+    u16
+    u32
+    u64
+    )
+
+set(NGRAPH_CPU_COMMON_DATATYPES
+    f32
+    i64
+    )
+
+
 if (NGRAPH_CPU_ENABLE)
     set(NGRAPH_CPU_DEBUGINFO_ENABLE 0 CACHE STRING "Enable debuginfo in the CPU backend")
 
+    set(NGRAPH_CPU_OPTIMIZED_DATATYPES "common"
```

Comment:
Going to hard-code it here instead of a cmake option ?

Reply:
These are cached variables. Will be set here only if they were not previously specified on the cmdline.